HotelBooking.API/Controllers/AddressController.cs

```diff
@@ -18,7 +17,7 @@ public AddressController(IAddressService addressService, ILogger<AddressControll
             this.logger = logger;
         }
 
-        [Authorize(Roles = "Administrator", AuthenticationSchemes = "Bearer")]
+        //[Authorize(Roles = "Administrator", AuthenticationSchemes = "Bearer")]
```

Comment:
Are there any issues with this? Why do you comment on this attr?

Reply:
Sorry! I want to test by swagger without authorization and authentication faster instead of using postman so I comment them.